packages/components/src/twin.d.ts

```diff
@@ -0,0 +1,28 @@
+import "twin.macro";
+import styledImport, { CSSProp, css as cssImport } from "styled-components";
+
+declare module "twin.macro" {
+  // The styled and css imports
+  const styled: typeof styledImport;
+  const css: typeof cssImport;
+}
+
+declare module "react" {
+  // The css prop
+  interface HTMLAttributes<T> extends DOMAttributes<T> {
+    css?: CSSProp;
+  }
+  // The inline svg css prop
+  interface SVGProps<T> extends SVGProps<SVGSVGElement> {
+    css?: CSSProp;
+  }
```

Comment:
Dang, adding a `css` attribute to all JSX elements is a bummer.

Any chance we can plan on not using those, and leave these types out? Or does that restrict the API too much?

Reply:
yeah after filling out the Type component, I feel pretty sure that we don't need it. will leave `styled` and `as` though